.ci/jenkins-execute-script.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
+
+import os
+import json
+import urllib
+import subprocess
+import sys
+
+url_base = "http://admin.ci.centos.org:8080"
```

Comment:
You need to use the cico client - do not call the api endpoint directly. Look at https://github.com/CentOS/python-cicoclient for more info.